src/MovieCard.js

```diff
@@ -12,16 +12,23 @@ import "./MovieCard.css"
 //     )
 // }
 
+//questions for kim: how do i use componenet did update
+//refactor functions to not have full fetch function inside of them
+//how to get back all the movies witrhout a re-fetch unless i have too
+
```

Comment:
Ah, I see my previous comments were already on your mind. Don't forget to clear comments before pushing up to github!